tests/unit/utils/test_qtutils.py

```diff
@@ -905,3 +907,4 @@ def test_double_exec(self):
         QTimer.singleShot(400, self.loop.quit)
         self.loop.exec_()
         assert not self.loop._executing
+    # pylint: enable=attribute-defined-outside-init
```

Comment:
No need for this, as pylint already only turns things off for this class and it's needed for the entire class.